sbws/lib/v3bwfile.py

```diff
@@ -43,6 +43,25 @@
 STATS_KEYVALUES = ['number_eligible_relays', 'minimum_number_eligible_relays',
                    'number_consensus_relays', 'percent_eligible_relays',
                    'minimum_percent_eligible_relays']
+BW_HEADER_KEYVALUES_RECENT_MEASUREMENTS_EXCLUDED = [
+    # The number of success results should be:
+    # the number of attempts - the number of failures
+    # 4.6 header: the number of successful results, created in the last 5 days,
+    # that were excluded by a filter
+    # This is the sum of the following 3 + not success results
+    # 'recent_measurement_exclusion_count',
+    'recent_measurement_exclusion_not_success_count',
+    # The relay measurements are not done in X time away from each other.
+    # By default this is 1 day.
+    'recent_measurement_exclusion_not_distanciated_count',
```

Comment:
Don't use two negatives "exclusion not distant", it's confusing.

Instead, explain what is being counted:

These recent measurements are in a short time range. Here is the exclusion count for those measurements.
```suggestion
    'recent_measurement_in_short_time_range_exclusion_count',
```